apache/zeppelin
# ZEPPELIN-3382 Installing interpreter from UI

### What is this PR for?
Installing a new interpreter from helium's UI. You just can click enable button to download a new interpreter you want to install

### What type of PR is it?
[Improvement]

### Todos
* [x] - Changed frontend side to enable users to click an install button
* [x] - Added status message for installation
* [x] - Added Backend logic to handle an interpreter installation

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3382

### How should this be tested?
1. Make sure you don't have `solr` interpreter in an interpreter create menu
1. Move to Helium/Interpreter menu and click the `solr` enable button
1. See the messages for downloading it and having downloaded it
1. Check interpreter creation menu to make sure `solr` exists

### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/3612566/38859396-e3548986-4268-11e8-9516-aeb1e87c85b3.png)
![image](https://user-images.githubusercontent.com/3612566/38859408-e9e66274-4268-11e8-8445-54d13b5c2a08.png)
![image](https://user-images.githubusercontent.com/3612566/38859588-50238418-4269-11e8-93d8-df8a125b0fcb.png)



### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Might be
